lib/services/sf.js

```diff
@@ -694,6 +695,17 @@ function StateAbstract(options)
             if (body && !body.success)
             {
               var data = body.data;
+              const auth = getAuthenticator(connectionConfig, httpClient);
+
+              if(body.code === '390195' && data.authnMethod === 'TOKEN') {
+                Logger.getInstance().trace("ID Token being used has expired. Reauthenticating");
+                const {CredentialManager} = require('../global_config');
+                CredentialManager.remove(connectionConfig.account, connectionConfig.username, authenticationTypes.ID_TOKEN_AUTHENTICATOR);
```

Comment:
After changes in global_config.js, here instead we can have:
```
let key = buildCredentialCacheKey(connectionConfig.account, connectionConfig.username, authenticationTypes.ID_TOKEN_AUTHENTICATOR); //this method can be put in `util.js` or wherever you think is appropriate :) 
GlobalConfig.getCredentialManager().remove(key)
```

Reply:
Fixed.